src/main/webapp/src/components/CourseDetails.vue

```diff
@@ -206,20 +253,117 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.label {
-    width: 7rem;
-    font-weight: 700 !important;
-    float: left !important;
+// page container
+.course-details-container {
+    margin: auto;
+    max-width: 1200px;
+}
+.course-details-main-container {
+    display: flex;
+    justify-content: space-between;
+}
+// buttons container
+.course-details-nav {
+    margin-bottom: $xxl-space;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.nav-item > button:first-child {
+    margin-right: $s-space;
+}
+.button-icon {
+    padding-right: $xs-space;
+}
+// course content
+.course-details-content-container {
+    max-width: 640px;
```

Comment:
magic width sollte vielleicht auch global standartisiert sein?

Reply:
ich glaube das ist nur für diese container spezifisch und würde nicht in anderen Platz benutzt

Reply:
habe ich ein container file erstellt wo alle Größe da sind.